libs/muddle/internal/muddle_register.hpp

```diff
@@ -115,13 +115,21 @@ class MuddleRegister : public network::AbstractConnectionRegister
   /// @}
 
 private:
+  using GaugePtr   = telemetry::GaugePtr<uint64_t>;
```

Comment:
`GaugePtr` doesn't look like it is being used

Reply:
ok